tutorials/actors_demo/actors_demo.md

```diff
@@ -0,0 +1,168 @@
+# Actors demo
+
+In this tutorial we will learn how to use `actors` to create a scripted animation.
+
+Animations are very useful if we want to have entities following a predefined path in simulation without being affected by the physics. This means they won't fall due to gravity or collide with other objects. They will however, have a 3D visualization which can be seen by RGB cameras, and 3D meshes which can be detected by GPU based sensors. The tutorial explains how to create open-loop trajectories which don't interact with the rest of the simulation.
+
+## Actors
+
+In Ignition Gazebo, an animated model is called an `actor`.
+
+There are two types of animations which can be used separately or combined together:
+
+* Skeleton animation, which is relative motion between links in one model:
+
+![skeleton_movement](skeleton_movement)
+
+* Trajectory animation, which carries all of the actor's links around the world as one group along a trajectory:
+
+![trajectory_movement](trajectory_movement)
+
+* Combined, to achieve a skeleton animation which moves in the world:
+
+![combined_movement](combined_movement)
+
+Actors have the following properties:
+
+* Actors are always static (i.e. no forces are applied on them, be it from gravity or contact or anything else)
```

Comment:
is this vaild? or it is not a model so also the concept of `static` is not there

Reply:
The text inside the parenthesis is true, but you're right that calling actors static doesn't make sense because they're not models.

So I'd remove the part about "static" but leave the information in the parenthesis.